client/package.json

```diff
@@ -9,6 +9,7 @@
     "@reduxjs/toolkit": "^1.6.0",
     "axios": "^0.21.1",
     "bootstrap": "^5.0.2",
+    "jwt-decode": "^3.1.2",
```

Comment:
Consider extending & using `/api/v1/users/current` endpoint to get all information about current user.  JWT may contain not all information required and there is no sence in storing avatar URL in JWT.

Reply:
this is for decode google token

Reply:
But a user may have dub-manager cookie set, so no login via google would be required.